### PR TITLE
Prerelease: 1.0.0-rc.11

### PR DESCRIPTION
**Pull request by kilchenmann**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## v1.0.0-rc.11 (15/09/2020)
+
+### Dependencies
+
+- [#234](https://github.com/dasch-swiss/dsp-js-lib/pull/234) This release is compatible with knora-api v13.0.0-rc.15
```
Comment (tobiasschweizer) [inline CHANGELOG.md:7]: Could you rephrase that to "Compatibility with knora-api v13.0.0-rc.15" (since it explains what the PR did)?

Comment (kilchenmann) [reply CHANGELOG.md:7]: I should use the title of the PR

Comment (tobiasschweizer) [reply CHANGELOG.md]: aha, ok

**Review by tobiasschweizer — APPROVED** [review]
